src/Services/Page.vala

```diff
@@ -78,6 +86,23 @@ public class ENotes.PageTable : DatabaseTable {
         }
 
         set_table_name ("Page");
+          
+        // @translator this code summarises a notebook page. Instead of given youtube link this code changes into Youtube Video      
+    
+        // Explaination for link: Regex for [Something](Something). As greedy as editor on markdown
+        var link = new BLMember(/\[[a-zA-Z0-9_\.\?\/:\=\+&\-'"]*\]\([a-zA-Z0-9_\.\?\/:\=\+&\-'"]*\)/, _("Link"));
```

Comment:
Looks like the first "something" doesn't work for text containing a space.

`[Link Text](https://webpage.com)` doesn't get changed to a link but `[LinkText](https://webpage.com)` does.

Reply:
and actually, What you should do is remove the `[ ](url)` instead of replacing the text with `link`, that way, if you wrote a text with a link on it, it won't look awkward with the replacement

Reply:
done